neo/renderer/RenderSystem.h

```diff
@@ -229,7 +229,7 @@ class idRenderSystem {
 	// This will perform swapbuffers, so it is NOT an approppriate way to
 	// generate image files that happen during gameplay, as for savegame
 	// markers.  Use WriteRender() instead.
-	virtual void			TakeScreenshot( int width, int height, const char *fileName, int samples, struct renderView_s *ref ) = 0;
+	virtual void			TakeScreenshot( int width, int height, const char *fileName, int samples, struct renderView_s *ref, int overrideFormat = -1 ) = 0;
```

Comment:
RenderSystem.h is part of the dhewm3-sdk, so changing a method signature here breaks the ABI - I wonder if we can avoid that somehow

Reply:
I resolved this by adding a global variable, `g_screenshotFormat`, that is `extern`'d in places where it's needed.